components/identities/actions/addEvmAddrModal.tsx

```diff
@@ -150,82 +155,70 @@ const AddEvmAddrModal: FunctionComponent<AddEvmAddrModalProps> = ({
+            onClick={() => closeModal(false)}
+          >
+            <svg viewBox="0 0 24 24">
+              <path
+                strokeLinecap="round"
+                strokeLinejoin="round"
+                strokeWidth="2"
+                d="M6 18L18 6M6 6l12 12"
+              ></path>
+            </svg>
+          </button>
+          <p className={styles.menu_subtitle}>Add an EVM address for</p>
+          <p className={styles.menu_title}>{identity?.domain}</p>
+          <div className="mt-5 flex flex-col justify-center">
+            <div className="mt-5">
+              <TextField
+                helperText={message ?? "Add your EVM address or your ENS name"}
+                fullWidth
+                label="Your EVM Address"
+                id="outlined-basic"
+                value={fieldInput ?? ""}
+                variant="outlined"
+                onChange={(e) => changeAddress(e.target.value)}
+                color="secondary"
+                required
+                error={!isValid}
+                InputProps={{
+                  endAdornment: (
+                    <InputAdornment position="end">
+                      {loading && <CircularProgress size={24} />}
+                    </InputAdornment>
+                  ),
+                }}
+              />
+            </div>
+            <div className="mt-5 flex justify-center">
+              <div>
+                <Button
+                  disabled={!evmAddress || !isValid}
+                  onClick={() => setUserData()}
+                >
+                  Set EVM address
+                </Button>
               </div>
-              <div className={styles.infoCard}>
-                <div>
-                  <h3 className={styles.cardTitle}>
-                    Why Add an EVM Address to Your Starknet Domain?
-                  </h3>
-                  <p className={styles.cardDesc}>
-                    By adding an EVM address to your Starknet domain, you
-                    enhance its functionality and connectivity. Your Starknet
-                    domain automatically comes with an associated ENS subdomain
-                    connected to all EVM chains and Rollups!
-                  </p>
-                </div>
-                <img
-                  src="/visuals/ecosystemMap.svg"
-                  className={styles.cardImg}
-                />
+            </div>
+            <div className={styles.infoCard}>
+              <div>
+                <h3 className={styles.cardTitle}>
+                  Why Add an EVM Address to Your Starknet Domain?
+                </h3>
+                <p className={styles.cardDesc}>
+                  By adding an EVM address to your Starknet domain, you enhance
+                  its functionality and connectivity. Your Starknet domain
+                  automatically comes with an associated ENS subdomain connected
+                  to all EVM chains and Rollups!
```

Comment:
By adding an EVM address to your Starknet domain, you can create a personalized ENS domain {name}.snid.eth. This domain can be used across all EVM chains and rollups ! Additionally, you'll update your ENS profile with your PFP, GitHub, Twitter, and other verified Starknet ID information!